corebehrt/main_causal/helper/select_cohort_full.py

```
@@ -301,3 +303,113 @@ def _ensure_stats_format(stats: dict) -> dict:
 
 def log_patient_num(logger: logging.Logger, df: pd.DataFrame, name: str):
     logger.info(f"N {name}: {len(df[PID_COL].unique())}")
+
+
+def split_and_save(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)